Expensify/App
# fix: App shows blank screen when user denies camera access

### Details
- The blank screen appears with missed handle errors when validating identity. Added 3 error messages that need to handle on iOS and Android platforms. 

### Fixed Issues

$ [#6841](https://github.com/Expensify/App/issues/6841)

### Tests

- Tested by denying camera access on iOS and Android.

### QA Steps

1. Go to the app
2. Send money to someone
3. Click on Continue button
4. Start verify identity
5. Deny access to camera
6. Go back then repeat step 4.
7. Open settings
8. Go back the app then verify 

### Tested On


### Screenshots

#### Web
N/A

#### Mobile Web
N/A

#### Desktop
N/A

#### iOS

https://user-images.githubusercontent.com/5449119/147815346-1f6f7cc5-29b7-4c30-b8e7-65fcbbb41791.mp4



#### Android


https://user-images.githubusercontent.com/5449119/147815353-a24ef5a2-6115-4565-92e3-56bd0bc331b5.mp4
